bitrepository-webclient/src/main/webapp/integrity-service.html

```diff
@@ -211,7 +207,7 @@ <h3 id="modalPagerLabel">Modal header</h3>
     function getPagingLimit(id, member) {
         var myID = id;
         var myMember = member;
-        return function () {
+        return function() {
             return pillars[myID][myMember];
```

Comment:
`pillars` was renamed to `pillarIntegrityStatuses` - this seems to be an issue in quite a few places

Reply:
Should be fixed now.